src/assignments/1-input-output.js

```diff
@@ -8,3 +8,11 @@ convert full name into uppercase
 print the 3rd character from users full name
 */
 // start here writing the code for program1
+
+const firstName = prompt("Enter first name");
+const lastName = prompt("Enter Last name");
+const fullName = firstName + lastName;
+console.log(fullName);
+console.log(fullName.length);
+console.log(fullName.toUpperCase());
+console.log(fullName[3]);
```

Comment:
third character should be : `fullName[2]`

Reply:
Where is the complete code on this.